lib/stm32/common/flash_common_all.c

```diff
@@ -21,7 +21,11 @@
 
 /**@{*/
 
+#ifndef GD32F1X0
 #include <libopencm3/stm32/flash.h>
+#else
```

Comment:
hrm, I'd kinda been thinking of just having the stm32/flash.h dispatch header include gd32 files, rather than having to chop into each of the c files like this too.  I guess this works as well. not really sure what's prettier.

Reply:
Agree.

Okay, thanks.